kotlin/authenticate-with-oauth/README.md

```diff
@@ -0,0 +1,37 @@
+# Authenticate with OAuth
+
+Authenticate with ArcGIS Online (or your own portal) using OAuth2 to access secured resources (such as private web maps or layers). Accessing secured items requires logging in to the portal that hosts them (an ArcGIS Online account, for example).
+
+![Image of authenticate with OAuth](authenticate-with-oauth.png)
+
+## Use case
+
+Your app may need to access items that are only shared with authorized users. For example, your organization may host private data layers or feature services that are only accessible to verified users. You may also need to take advantage of premium ArcGIS Online services, such as geocoding or routing services, which require a named user login.
+
+## How to use the sample
+
+When you run the sample, the app will load a web map which contains premium content. You will be challenged for an ArcGIS Online login to view the private layers. Enter a user name and password for an ArcGIS Online named user account (such as your ArcGIS for Developers account). If you authenticate successfully, the traffic layer will display, otherwise the map will contain only the public basemap layer.
+
+## How it works
+
+1. Set the `AuthenticationManager`'s `AuthenticationChallengeHandler` to a `DefaultAuthenticationChallengeHandler`.
+1. Create an `OAuthConfiguration` specifying the portal URL, client ID, and redirect URL.
+1. Add the OAuth configuration to the authentication manager.
+1. Load a map with premium content requiring authentication to automatically invoke the default authentication handler.
+
+## Relevant API
+
+ * AuthenticationManager
```

Comment:
I think I'm on the most up to date version of this branch, but I'm still seeing these bullet points as stepped and not inline as mentioned by Trevor